packages/opentelemetry-core/test/trace/tracestate.test.ts

```diff
@@ -79,9 +79,23 @@ describe('TraceState', () => {
       assert.deepStrictEqual(state.serialize(), 'a=1,c=3');
     });
 
-    it('must parse states that only have a single value with an equal sign', () => {
+    it('must skip states that only have a single value with an equal sign', () => {
```

Comment:
Does the spec explicitly state to not accept `=` in the value? It sounds like this could be problematic for Base64.

Reply:
"Value is opaque string up to 256 characters printable ASCII [RFC0020] characters (i.e., the range 0x20 to 0x7E) except comma , and =", this is what is mentioned in the specs: https://w3c.github.io/trace-context/#header-value

Reply:
What is the recommended format since the spec explicitly blocks the use of Base64?

Reply:
Maybe URL encoded?